src/ConnectCore.js

```diff
@@ -87,6 +88,31 @@ class ConnectCore {
     // TODO throw error if this.network not part of network set in Credentials
     this.canSign = !!this.credentials.settings.signer && !!this.credentials.settings.address
     this.pushToken = null
+    this.PubSub = PubSub
+
+    this.loadPlugins()
+  }
+
+  loadPlugins() {
```

Comment:
don't disagree, but right now I believe there is no overhead in size and or end user complexity, if you don't care about the buttons you don't need to know anything about them and it doesn't add any large dependencies. But expecting that we may also create different buttons and plugins it may likely be worthwhile placing them in a separate class. The only thing is if we want to provide  a different dist files for buttons or provide a single entry point, just have decide how to package this all up. Also the code is not only in the classes but some is at the top level when the lib file is loaded